src/plugins/newGuildSettings/index.tsx

```diff
@@ -77,8 +96,11 @@ export default definePlugin({
         updateGuildNotificationSettings(guildId,
             {
                 muted: settings.store.guild,
+                message_notifications: settings.store.messages,
                 suppress_everyone: settings.store.everyone,
-                suppress_roles: settings.store.role
+                suppress_roles: settings.store.role,
+                mute_scheduled_events: settings.store.events,
+                notify_highlights: settings.store.highlights ? 1 : 2
```

Comment:
wouldn't this make more sense as HighlightSettings.NULL? I assume thats the default value
```suggestion
                notify_highlights: settings.store.highlights ? 1 : 0
```

Reply:
Just checked again and you're right, NULL is the default value, don't know why I used that tbh